app/konnect/network.md

```diff
@@ -8,12 +8,12 @@ The {{site.konnect_saas}} control plane uses the following ports:
 
 | Port      | Protocol  | Description |
 |:----------|:----------|:------------|
-| `443`    | TCP <br>HTTPS | Cluster communication port for configuration and telemetry data. The {{site.konnect_saas}} control plane uses this port to listen for runtime node connections and to communicate with the runtime nodes. <br> The cluster communication port must be accessible by all the data planes within the same cluster. This port is protected by mTLS to ensure end-to-end security and integrity. |
+| `443`    | TCP <br>HTTPS | Cluster communication port for configuration and telemetry data. The {{site.konnect_saas}} control plane uses this port to listen for connections and communicate with data planes.. <br> The cluster communication port must be accessible by all the data planes within the same cluster. This port is protected by mTLS to ensure end-to-end security and integrity. |
```

Comment:
```suggestion
| `443`    | TCP <br>HTTPS | Cluster communication port for configuration and telemetry data. The {{site.konnect_saas}} control plane uses this port to listen for connections and communicate with data planes. <br> The cluster communication port must be accessible by all the data planes within the same cluster. This port is protected by mTLS to ensure end-to-end security and integrity. |
```
I thought the nodes were the ones that communicated using the port instead of the data plane?

Reply:
Yep, let's rephrase all of this to say "data plane nodes" if we're referring to individual gateway instances.